cypress/tsconfig.json

```diff
@@ -1,7 +1,5 @@
 {
```

Comment:
Do you know where this tsconfig is actually used?

Reply:
oops i didn't mean to change this, i'll remove changes. but it gets used locally when you run cypress